frontend/packages/topology/src/components/side-bar/components/SideBarAlerts.tsx

```diff
@@ -16,14 +17,19 @@ const ResolveResourceAlerts: React.FC<{
   useResourceAlertsContent?: (element: GraphElement) => DetailsResourceAlertContent;
   element: GraphElement;
 }> = observer(function ResolveResourceAlerts({ id, useResourceAlertsContent, element }) {
-  const [showAlert, setShowAlert, loaded] = useUserSettings(
+  const [showAlertFromConfigMap, , loaded] = useUserSettings(
     `${USERSETTINGS_PREFIX}.${SIDEBAR_ALERTS}.${id}.${element.getId()}`,
     true,
   );
+  const [showAlert, setShowAlert] = useUserSettingsLocalStorage(
+    `${USERSETTINGS_PREFIX}/${SIDEBAR_ALERTS}/${id}`,
+    `${element.getId()}`,
+    true,
```

Comment:
Will defaulting to true here override the value from localStorage if there is no value set in the config map? I'd think this would break disabling the alert.